src/main/java/fi/aalto/cs/apluscourses/ui/AboutDialog.java

```diff
@@ -58,11 +83,52 @@ private static JPanel createAboutInnerContainer() {
     return linkLabel;
   }
 
+  @NotNull
+  private static LinkLabel<Object> createAPlusWebsiteLinkLabel() {
+    LinkLabel<Object> linkLabel = new LinkLabel<>(
+            getText("ui.aboutDialog.APlusWebsite"),
+            AllIcons.Ide.External_link_arrow,
+            (first, second) ->
+                    BrowserUtil.browse(A_PLUS_PAGE));
```

Comment:
question to @Aalto-LeTech/idea-plugin-dev:
was this something you meant when discussing the #403

Reply:
Yes, and `BrowserUtil.browse` worked, whereas the standard Java option of `Desktop.browse` did not, although that student had some obscure setup so we don't need to be very concerned with that issue